### ISSUE TITLE
Recommend only using 3 points for z_tilt

### ISSUE THREAD

**Issue by dancernogorsky** [heschlie/vc3-klipper-configs#2]
https://github.com/heschlie/vc3-klipper-configs/blob/9d985681fd949db97c7213696b35955fca357ad7/printer.cfg#L364-L370

I was referencing your config (thank you) on my 500 and could not get my tolerance < .05 when performing a z tilt.

After logging and a bunch of tail chasing, I figured out that have > 3 points caused the the x tilt adjustment calculation to fight itself.

If you remove just one of the rear points and center the other rear point, you will be able to remove 'retry_tolerance' and use the default .02 value and your bed will be better aligned.

Cheers!

**Comment by heschlie** [comment]
Thanks, but I specifically picked those points as my bed has a big high spot right where the rear Z screw is which causes the Z tilt adjust to actually leave the bed tilted, if it is ever able to get within tolerance. The 4 points I picked on my bed are pretty much coplanar so once it is within tolerance it is pretty flat. I have printed a few prints that use the entire bed without any issue with these settings so I'm quite happy with them.

This is one of the reasons I put "These configs are specific to my setup" in the README and in the configs, I don't expect them to work for everyone as I'm compensating for quirks in my machine :)
